Uppsala/UppEt3.xml

```diff
@@ -0,0 +1,557 @@
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">80</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>154</height>
+                                        <width>112</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="16 17">
+                                    <dimensions unit="mm" xml:id="textarea1">
+                                        <height>97</height>
+                                        <width>75</width>
+                                    </dimensions>
+                                    <note>Data on dimensions have been adopted from the entry on the manuscript in
+                                        <bibl><ptr target="bm:alvin"/></bibl>.
+                                    </note>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">31</citedRange>
+                                </bibl> calls it a "large and even archaic script" 
+                                    (<foreign xml:lang="de">Große and gleichmäßige archaische Schrift</foreign>).</desc>
+                                <date notBefore="1400" notAfter="1499">15th century.</date>
+                            </handNote>
+                        </handDesc>
+                        
+                        <decoDesc>
+                            
+                            <decoNote xml:id="d1" type="Headband">
+                                <locus target="1r"/>
+                                <desc>Ornamental band (braided pattern) in black and red on top of the text.</desc>
+                            </decoNote>
+                            
+                        </decoDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="55r"/>
+                                    <desc type="Unclear">Under the supplication to <ref target="#ms_i1"/>, between the
+                                        <foreign xml:lang="la">cruces ansatae</foreign>, the remnants of an erased
+                                        note are visible. Only one word has been left untouched. The note is not mentioned by 
+                                        <bibl><ptr target="bm:Zettersteen1899Upsala"/></bibl> or
```

Comment:
I do not think this info is necessary (that someone does not mention smth).

Reply:
But if the default is that the descriptions is based on the catalogues mentioned at the end, then there needs to be some way of marking what we add, no?

Reply:
Yes, this is a complicated issue, but you will soon find it is impossible to come through if you want to be very exact. We say: we base our descriptions on smth, and this implicates that we as cataloguers take the old foundation and bring smth new/improve older descriptions, as our routine work. If there is smth very important, you can make a note in the edition history.

Reply:
Ok, yes, I will certainly learn with time :) Thank you for the input on this!